rio_cogeo/cogeo.py

```diff
@@ -242,6 +242,17 @@ def cog_translate(
                         rasterio.open(tmpfile.name, "w", **meta)
                     )
 
+                # Transfer color interpolation
+                ci = vrt_dst.colorinterp
+                if nodata is None and not alpha:
+                    ci = [c for c in ci if c != ColorInterp.alpha]
+                if indexes:
+                    if len(indexes) == 1:
```

Comment:
let's add
```python
    if isinstance(indexes, int):
        indexes = (indexes,)
```
at the beginning of the script (L118) so indexes is always a tuple or a list